docs/sources/installation/istio.md

```diff
@@ -0,0 +1,179 @@
+# Installation instructions for Istio
```

Comment:
This file is missing the page metadata, and thus appears as a blank space in the left navigation/table of contents.  If we want people to be able to find this topic, the following added to the top of the file:

---
title:  Istio service mesh
weight: 45
---